Wrappers.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include "City.h"
+#include "ControlRoom.h"
+#include "LiveRoom.h"
+#include "StudioComplex.h"
+#include "LargestFiveCities.h"
```

Comment:
Don't include what you don't use.
None of the classes below are USING the pointers in this header file. 
That means you can forward declare all of these types.
Include these header files where the types are actually used: in the cpp file.
- [ ] revise